puma/hlplots/tagger.py

```diff
@@ -0,0 +1,214 @@
+"""Tagger module for high level API."""
+import h5py
+import numpy as np
+import pandas as pd
+from numpy.lib.recfunctions import structured_to_unstructured
+
+from puma.utils import calc_disc, logger
+
+
+class TaggerBase:  # pylint: disable=too-many-instance-attributes
+    """Base class handling tagger results."""
+
+    def __init__(self, model_name: str) -> None:
+        """Init Tagger class.
+
+        Parameters
+        ----------
+        model_name : str
+            Name of the model, also correspondinng to the pre-fix of the tagger
+            variables.
+        """
+        self.model_name = model_name
+        self.label = None
+        self.reference = False
+
+        self.scores = None
+        self.perf_var = None
+        self.flvs = ["pu", "pc", "pb"]
```

Comment:
something like `output_nodes` would be a more intuitive attribute name in my opinion, since `flvs`=?`flavours` doesn't directly tell me that it's the probabilities/predictions we are talking about here.

Reply:
I fixed that